package ExceptionHandling;

public class Test1
{
    int a=10;
   static int b=30;
   static int c=0;
    public static void main(String[] args) {
        //  System.out.println("divide by 0:"+(b/c));
        ThrowExcep obj = new ThrowExcep();//throw
        c obj1=new c();//throws
//obj1.cb();//throws

        obj.fun();//throw
//        try{
//            Test1 obj=new Test1();//ArithmeticException
//            int result =b/c;
//            System.out.println(" exception handle:"+result);
//
//        }
//        catch(RuntimeException e)
//        {
//            System.out.println("can't divide by o: ");
//        }
//
//    }
    }
}
